<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_FlaSetAdvancedPropertiesWidget</class>
 <widget class="QWidget" name="C_FlaSetAdvancedPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_FlaSetAdvancedPropertiesWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgeWiSettingSubSection" name="pc_WiHeader" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="pc_WiContent" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="2" column="0">
       <widget class="stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabTransferDataTimeout">
        <property name="text">
         <string>Transfer Data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="stw::opensyde_gui_elements::C_FlaOgeSpxDark" name="pc_SpxTransferDataTimeout">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabFlashloaderResetWaitTime">
        <property name="text">
         <string>Wait Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabRequestDownloadTimeout">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="stw::opensyde_gui_elements::C_FlaOgeSpxDark" name="pc_SpxRequestDownloadTimeout">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="stw::opensyde_gui_elements::C_FlaOgeSpxDark" name="pc_SpxFlashloaderResetWaitTime">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgeWiSettingSubSection</class>
   <extends>QWidget</extends>
   <header>C_CamOgeWiSettingSubSection.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGenericNoPaddingNoMargins.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_FlaOgeSpxDark</class>
   <extends>QSpinBox</extends>
   <header>C_FlaOgeSpxDark.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
